src/ProtocolStreamReader.php

```diff
@@ -30,33 +30,32 @@ public function __construct($input)
     {
         $this->input = $input;
         Loop\addReadStream($this->input, function() {
-            while (($c = fgetc($this->input)) !== false) {
-                $this->buffer .= $c;
-                switch ($this->parsingMode) {
-                    case ParsingMode::HEADERS:
-                        if ($this->buffer === "\r\n") {
-                            $this->parsingMode = ParsingMode::BODY;
-                            $this->contentLength = (int)$this->headers['Content-Length'];
-                            $this->buffer = '';
-                        } else if (substr($this->buffer, -2) === "\r\n") {
-                            $parts = explode(':', $this->buffer);
-                            $this->headers[$parts[0]] = trim($parts[1]);
-                            $this->buffer = '';
+            $c = fgetc($this->input);
```

Comment:
I opened a [PR for the sabre project](https://github.com/fruux/sabre-event/pull/40).
This doesn't fix the windows problem, however. Maybe we need an OS switch in the extension.js.